Wormhole potion and wormhole mirror error

Using either of the two dc's you with an Internal Server Error. I don't currently have access to the Log but will try to get it later. 

Haven't tested singleplayer. 
Pack: Enigmatica 4
bountifulbaubles-1.14.4-0.0.2
forge: 28.1.87

Serverside Error: 
```[30Nov2019 20:57:44.850] [Server thread/WARN] [net.minecraft.network.NetworkSystem/]: Failed to handle packet for /84.142.221.107:62764
net.minecraft.crash.ReportedException: Ticking player
	at net.minecraft.entity.player.ServerPlayerEntity.func_71127_g(ServerPlayerEntity.java:428) ~[?:?]
	at net.minecraft.network.play.ServerPlayNetHandler.func_73660_a(ServerPlayNetHandler.java:183) ~[?:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:228) ~[?:?]
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:135) [?:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:846) [?:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:324) [?:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:764) [?:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:622) [?:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_201]
Caused by: java.lang.NullPointerException: A given capability is null
	at java.util.Objects.requireNonNull(Unknown Source) ~[?:1.8.0_201]
	at org.cyclops.cyclopscore.modcompat.capabilities.DefaultCapabilityProvider.getCapability(DefaultCapabilityProvider.java:42) ~[?:1.5.5]
	at net.minecraftforge.common.capabilities.CapabilityDispatcher.getCapability(CapabilityDispatcher.java:102) ~[?:?]
	at net.minecraftforge.common.capabilities.CapabilityProvider.getCapability(CapabilityProvider.java:118) ~[?:?]
	at net.minecraft.entity.LivingEntity.getCapability(LivingEntity.java:2890) ~[?:?]
	at net.minecraft.entity.player.PlayerEntity.getCapability(PlayerEntity.java:2106) ~[?:?]
	at cursedflames.bountifulbaubles.common.wormhole.ContainerWormhole.<init>(ContainerWormhole.java:56) ~[?:?]
	at cursedflames.bountifulbaubles.common.item.items.ItemPotionWormhole$1.createMenu(ItemPotionWormhole.java:68) ~[?:?]
	at net.minecraftforge.fml.network.NetworkHooks.openGui(NetworkHooks.java:188) ~[?:?]
	at net.minecraftforge.fml.network.NetworkHooks.openGui(NetworkHooks.java:135) ~[?:?]
	at cursedflames.bountifulbaubles.common.item.items.ItemPotionWormhole.doWormhole(ItemPotionWormhole.java:77) ~[?:?]
	at cursedflames.bountifulbaubles.common.item.items.ItemPotionWormhole.func_77654_b(ItemPotionWormhole.java:85) ~[?:?]
	at net.minecraft.item.ItemStack.func_77950_b(ItemStack.java:196) ~[?:?]
	at net.minecraft.entity.LivingEntity.func_71036_o(LivingEntity.java:2570) ~[?:?]
	at net.minecraft.entity.player.ServerPlayerEntity.func_71036_o(ServerPlayerEntity.java:981) ~[?:?]
	at net.minecraft.entity.LivingEntity.func_184608_ct(LivingEntity.java:2462) ~[?:?]
	at net.minecraft.entity.LivingEntity.func_70071_h_(LivingEntity.java:1965) ~[?:?]
	at net.minecraft.entity.player.PlayerEntity.func_70071_h_(PlayerEntity.java:232) ~[?:?]
	at net.minecraft.entity.player.ServerPlayerEntity.func_71127_g(ServerPlayerEntity.java:365) ~[?:?]
	... 8 more
[30Nov2019 20:57:44.904] [Server thread/INFO] [net.minecraft.network.play.ServerPlayNetHandler/]: Darkere lost connection: Internal server error
[30Nov2019 20:57:44.904] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Darkere left the game```

--- a/build.gradle
+++ b/build.gradle
@@ -12,19 +12,17 @@ buildscript {
     dependencies {
         classpath 'net.minecraftforge.gradle:ForgeGradle:2.3-SNAPSHOT'
 		classpath 'org.spongepowered:mixingradle:0.4-SNAPSHOT'
-		classpath "com.wynprice.cursemaven:CurseMaven:1.1.0"
     }
 }
 apply plugin: 'net.minecraftforge.gradle.forge'
 apply plugin: 'java'
-apply plugin: "com.wynprice.cursemaven"
 
 reobf {
 	
 }
 
 
-version = "1.12.2-0.1.4"
+version = "1.12.2-0.1.5-wormhole"
 group = "cursedflames.bountifulbaubles" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
 archivesBaseName = "Bountiful Baubles"
 
@@ -84,7 +82,7 @@ dependencies {
     //compile "some.group:artifact:version:classifier"
     //compile "some.group:artifact:version"
 	compile "baubles:Baubles:1.12:1.5.2"
-	deobfCompile curse.resolve("albedo", "2714505")
+	deobfCompile "albedo:albedo:1.12.2:1.1.0"
 	
 	// compile against the JEI API but do not include it at runtime
 	deobfProvided "mezz.jei:jei_1.12.2:4.12.1.217:api"

--- a/build.gradle
+++ b/build.gradle
@@ -22,7 +22,7 @@ reobf {
 }
 
 
-version = "1.12.2-0.1.5-wormhole"
+version = "1.12.2-0.1.4"
 group = "cursedflames.bountifulbaubles" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
 archivesBaseName = "Bountiful Baubles"
 

--- a/src/main/java/cursedflames/bountifulbaubles/wormhole/ContainerWormhole.java
+++ b/src/main/java/cursedflames/bountifulbaubles/wormhole/ContainerWormhole.java
@@ -116,6 +116,8 @@ public class ContainerWormhole extends Container {
 		changes.setInteger("pinCount", pinCount);
 //		BountifulBaubles.logger.info(changes);
 		for (int j = 0; j<this.listeners.size(); ++j) {
+			if (!(this.listeners.get(j) instanceof EntityPlayerMP))
+				continue;
 			PacketHandler.INSTANCE.sendTo(
 					new NBTPacket(changes, PacketHandler.HandlerIds.WORMHOLE_UPDATE_GUI.id),
 					((EntityPlayerMP) this.listeners.get(j)));

